### ISSUE TITLE
Improve text colour contrast to improve accessibility

### ISSUE THREAD

**Issue by ekafyi** [kawalcovid19/wargabantuwarga.com#572]
## Description

Several text/link elements in our Home page have low colour contrast:
- "Informasi Terbaru" dates
- "Informasi Terbaru" CTA (eg. _"Daftar vaksin Sinergi Sehat"_)
- Whatsapp contact CTA

See: ["Accessibility" section in the Lighthouse Report](https://lighthouse-dot-webdotdevsite.appspot.com//lh/html?url=https%3A%2F%2Fwww.wargabantuwarga.com%2F#accessibility)

## Expected Behaviour

The text/link elements should have a contrast of at least `4.5`.

---

First-time contributor? See contributing guide here ([English](https://github.com/kawalcovid19/wargabantuwarga.com/blob/main/CONTRIBUTING.md) / [Bahasa Indonesia](https://github.com/kawalcovid19/wargabantuwarga.com/blob/main/CONTRIBUTING_ID.md)).
